.github/workflows/ks-staging-env-acceptance-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: Kesäseteli Staging Environment Acceptance tests
+# Disabled until staging is not behind vpn anymore
+# on:
+  # schedule:
+    # At half past every hour from 7-20 (finnish winter time)
+    # - cron: "*/30 5-18 * * *"
+env:
+  FRONTEND_URL: https://yjdh-kesaseteli-ui-stg.apps.platta.hel.fi
+  BRANCH: main
+jobs:
+  acceptance-tests:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    name: Acceptance tests for Test Environment
```

Comment:
Acceptance tests for Staging Environment

Reply:
fixed